package.json

```diff
@@ -37,5 +37,9 @@
     "tmp-promise": "^3.0.3",
     "vitest": "^1.6.0"
   },
-  "packageManager": "pnpm@9.5.0"
+  "packageManager": "pnpm@9.5.0",
+  "volta": {
+    "node": "22.9.0",
```

Comment:
remove this, we can have arguments about volta for pnpm but if you want to pin the node version use the `.npmrc` method so that people not using volta would have the same experience

Reply:
that only works if I'm also using corepack to choose the version of pnpm that switches both node and pnpm version :-\